app/views/travel_sponsorships/index.html.haml

```diff
@@ -5,6 +5,7 @@
 .navbar
   .navbar-inner
     = search_form_for @q, :class => "navbar-form pull-right" do |f|
+      = f.text_field(:user_nickname_or_event_name_or_user_profile_full_name_cont, :placeholder => "Nickname or Conference Title or Full Name", :class=>"input-xlarge")
```

Comment:
Done. In the placeholder, I have changed 'Conference Title' to 'Event name' to maintain consistency.